Samples/Samples.Android/Properties/AndroidManifest.xml

```diff
@@ -8,7 +8,9 @@
 	<uses-permission android:name="android.permission.CAMERA" />
 	<uses-permission android:name="android.permission.FLASHLIGHT" />
 	<uses-permission android:name="android.permission.INTERNET" />
+  <uses-permission android:name="android.permission.READ_CALENDAR" />
```

Comment:
Can we move the added permissions so that they're next to each other

Reply:
Ahh I see, it lines up when you edit it in xml but the difference is spaces vs tabs, fixing now.

Reply:
No I meant move read and write permissions to consecutive lines

Reply:
Sure, I only had them that way as they seemed to be listing permissions in alphabetical order

Reply:
ah good point, I hadn't picked up on that. Leave as is then